### DISCUSSION TITLE
Moving discussions to community.ansel.photos

### DISCUSSION THREAD

**Discussion by aurelienpierre** [aurelienpierreeng/ansel#131] [Announcements]
Github is unwelcoming for non-programmers and non-english-speakers. I opened a social platform on https://community.ansel.photos. The benefit of it is to allow image sharing and the underlying content management software can be extended with online learning features like courses, classes and quizzes.

Discussions here will be closed in the next week.

**Comment by llajtamasi** [comment]
Further to my previous comment (https://github.com/aurelienpierreeng/ansel/discussions/21#discussioncomment-5341739), I have now compiled and reviewed Ansel a bit further.  It compiled fine, the software works, its smaller than DT, I don't find it any more intuitive than DT, its development appears to be governed by an individual model. Some ironies to me...
1. the main reason put forward for the fork is that DT is run by computer people and not by photographers and the main changes are computer programming changes that the  Ansel developer thinks are better than the DT community decisions. DT may be bloated, but my car is also bloated (eg. I've never used and will never use the cigarette lighter) but I won't ditch a recognised global model with repair capabilities available everywhere with a model by an individual engineer who does not like cigarette lighters.
2. the rallying cry of free/libre, "anything goes", tolerating all language and material however offensive to the general populace and  governed by French law when tens of thousands paid for freedom at the guillotine blade seems ironic. Asserting your right to be an offensive mongrel and spurning all the other "idiots" seems gratuitously indulgent.

I took a snapshot of my DT image storage (35K+ images) and loaded them into Ansel and tried it for a few days. In the end, I'm going back to DT. It appears a more civil community with a broader view point. We have enough autocratic dictators in the world. I wish the new Ansel community well - there is a lot of technical ability in its leader to develop the platform.

**Reply by aurelienpierre** [reply]
I think you didn't got the fact that darktable's maintenance costs are skyrocketting, bugs are getting weirder and weirder, and hidden within thousands of new lines added over the past years for little to no user benefit, no matter how "repairable" you think the randomized development model is.

Also the Ansel GUI is undergoing refactoring (found the menu with shortcuts ?) and many silent bugs and GUI inconsistencies have been "fixed" by removing special cases handled through crazy branching, because the 5 overlays × 7 + 1 lighttable flavours led to 36 combinations of thumbnails rendering with several untested pathes.

Ansel is run by the only dev who bothered to teach users how to use it, either on YouTube or in private lessons, and the design changes are motivated by me tired of repeating the same things over and over again because even the people who understand the theory still don't find their ducks.

Take from that what you will, I don't care. Darktable democracy has become a mediocracy of amateur devs fucking over theory they don't grasp to take decisions leading to overcomplicated design at every step.

To conclude : yes, people with modest hardware have reported substantial speed-ups.

**Reply by lromano** [reply]
I'm personally pretty happy to see this new proposal arrive.

As a user, I began to think that Darktable was capable enough, and that perhaps the moment had come to pause, consider what had been accomplished, and simplify the product.
Aurélien is proposing something new and I'm glad he is.
I'm curious to test and follow up changes.

Being myself a developer, I can totally relate to the purpose of refactoring and code cleanup because it facilitates maintanability, reduces chances of bugs and eases new features addition.

Also, making ajustements to the user experience (UI + performances) is certainly beneficial for the users and for a larger adoption.

Even though its mostly a one-guy effort , At least, its a clear proposal and a 'living' exemple of a possible better direction.
Time will tell if its will outweight  its 'parent' project, but I personally adhere to the motivations, really relevant.
